routes/web.php

```diff
@@ -34,17 +38,14 @@
 })->name('verification');
 
 Route::middleware(['auth', 'log'])->group(function () {
-    Route::get('/home', 'HomeController@index')->name('home');
-    Route::get('/user', 'UserController@index')->name('user');
     Route::get('/test_mails/{mail}/{send?}', 'EmailController@testEmail');
 });
 
-Route::middleware(['auth', 'verified'])->group(function () {
-    Route::post('userdata/update_password', 'UserController@updatePassword')->name('userdata.update_password');
-});
-
 Route::middleware(['auth', 'log', 'verified'])->group(function () {
+    Route::get('/home', 'HomeController@index')->name('home');
     Route::get('userdata', 'UserController@showData')->name('userdata');
+    Route::get('/user', 'UserController@index')->name('user');
+    Route::post('userdata/update_password', 'UserController@updatePassword')->name('userdata.update_password');
```

Comment:
@luksan47 You said updating the password should not be logged.

Reply:
Right, that was a mistake, corrected with the merge commit.